packages/client/hmi-client/src/components/workflow/ops/model-config/tera-model-config-drilldown.vue

```diff
@@ -628,10 +628,6 @@ const resetConfiguration = () => {
 	});
 };
 
-onMounted(async () => {
```

Comment:
Since we have `immediate:true` on the watcher, `initialize()` was always being called _twice_.  Now initialize will always be called when the component is created and whenever then output changes _one_ time